src/app/component-library/catalog/asset-detail-dialog/asset-detail-dialog.component.ts

```diff
@@ -40,10 +40,14 @@ export class AssetDetailDialogComponent implements OnDestroy {
   loading = false;
 
   get negotiationState(): 'ready' | 'negotiating' | 'negotiated' {
-    const contractOffer = this.data.contractOffer!;
-    if (this.contractNegotiationService.isNegotiated(contractOffer)) {
+    const dataOffer = this.data.dataOffer!;
+    if (
+      this.contractNegotiationService.isNegotiated(dataOffer.contractOffers[0])
+    ) {
```

Comment:
if (this.contractNegotiationService.isNegotiated(contractOffer)) {
...
} looks like a better format